src/main/resources/com/atlassian/jira/cloud/jenkins/configuration/config.jelly

```diff
@@ -79,7 +79,7 @@
 
                         <div id="siteDataContainer" class="site-data-default" name="sites">
                             <f:section title="Site Connection details">
-                                <div class="help-area">
```

Comment:
Was unused here so ripped it out

Reply:
It's hard to see in GH's editor but do we need the div at all? It looks like there's one just below it and a section above it so I wonder whether it's doing anything useful

Reply:
Nice catch, its been destroyed